rtv/subreddit.py

```diff
@@ -89,6 +91,41 @@ def open_link(self):
         global opened_links
         opened_links.add(url)
 
+    @SubredditController.register('p')
+    def post_submission(self):
+        # Abort if user isn't logged in
+        if not self.reddit.is_logged_in():
+            show_notification(self.stdscr, ["Login to reply"])
+            return
+
+        name = self.content.name
+        if '+' in name:
```

Comment:
Yes it will fail.

```python
try:
    sub = self.reddit.get_subreddit(name)
except APIException:
    # Handle case where subreddit doesn't exist
```
Go with your way if you think it's cleaner though. Also don't forget about the modifiers like ``/r/front/new`` and ``/r/all/controverisal``